### ISSUE TITLE
Where can I find some documentation about URL parameter?

### ISSUE THREAD

**Issue by aborruso** [githubocto/flat-viewer#18]
Hi,
when I apply a filter I see something like:

```
?filename=dati%2Fsomministrazioni-vaccini-latest.csv
&filters=fornitore%3DJanssen
&sha=9e6a9f4a150e6b96ef52827e0e2692274d68701e
&sort=data_somministrazione%2Cdesc
&stickyColumnName=data_somministrazione
```

For some I can deduce the meaning. Is there a page with documentation? 
What's a stickyColumnName, what is it for?
What's the date format I must use for date fields? etc.

Thank you

**Comment by RandomFractals** [comment]
I second that request. Your flat data viewer has some nifty params to pull data from commits, etc. Would be good if they were documented in this repo for us to explore data files with different url params to create and share custom data URL views.

Thanks!

**Comment by Wattenberger** [comment]
The url params mirror the current state of the table, for example the "stickyColumnName" represents the leftmost-column, which sticks when you scroll right. For creating and sharing URLs, you should be able to just filter and sort as you please, then share the current url - way easier than URL hacking.

I'd love to hear about any use cases where that's not an option, though!

**Comment by RandomFractals** [comment]
yeah, but I think you can create a simple md table in docs to document those parts.

**Comment by aborruso** [comment]
> I'd love to hear about any use cases where that's not an option, though!

A simple one: use fully the filters.

An example. [Here](https://flatgithub.com/italia/covid19-opendata-vaccini?filename=dati%2Fsomministrazioni-vaccini-latest.csv&sha=57fddf470f2f4b9a95ee5c020aec59dcaf908161) I would like to filter date from 2021-03-21 to 2021-04-05. Using only the GUI I'm not able to do it. But using and understanding URL parameters, having a URL parameters README I could build a right URL, the URL I need, using unixtimestamp `&filters=<myfield>=<start>,<stop>`.

The same using filter on `codice_regione_ISTAT`. I'm not able via GUI to filter only the `19` region. Once again having a documentation I could write `&filters=codice_regione_ISTAT=19,19`.

![image](https://user-images.githubusercontent.com/30607/120498400-7a1fb400-c3bf-11eb-9b03-9d689711589d.png)

Thank you

**Comment by RandomFractals** [comment]
@aborruso that's a perfect example. I was also looking for something like that param line example you provided but in md table that describes each param function and uses similar syntax for examples in description column.

does not have to be fancy

| Parameter Name | Parameter Values | Description |
| --- | --- | --- |
| `filters` | `<field1>,<field2>` | Filter to apply to the flat data view |

...

**Comment by aborruso** [comment]
Yes @RandomFractals . I think it would be very useful to add the value type: in example for field mapped as date it's necessary to use Unix timestamp, etc.

**Comment by RandomFractals** [comment]
right, so, I think supported data types and filter data formats should be listed too. good point!

**Comment by Wattenberger** [comment]
Since this isn't supported functionality, it's important to be careful how we document "url-hacking". However, I can see that documentation being useful, if either of you wants to take a stab at it.

The relevant code lives in these two places:

https://github.com/githubocto/flat-viewer/blob/main/src/components/repo-detail.tsx#L39-L40
https://github.com/githubocto/flat-viewer/blob/main/src/components/json-detail-container.tsx#L24-L30

**Comment by RandomFractals** [comment]
I'll let @aborruso take a stab at it. Good to know where to look tho. Thanks for those code links.

btw, I was going over your open requests. Added this bit for folks looking for charts for now: https://github.com/githubocto/flat-viewer/issues/17#issuecomment-852998337
